x-pack/plugins/apm/public/context/environments_context/environments_context.tsx

```diff
@@ -36,8 +41,14 @@ export function EnvironmentsContextProvider({
     ('environment' in query && (query.environment as Environment)) ||
     ENVIRONMENT_ALL.value;
 
-  const rangeFrom = 'rangeFrom' in query ? query.rangeFrom : undefined;
-  const rangeTo = 'rangeTo' in query ? query.rangeTo : undefined;
+  /* eslint-disable prettier/prettier */
```

Comment:
is it really necessary?

Reply:
removed in favour of your next comment. Thank you for the suggestion